### ISSUE TITLE
alphabetize dependencies/devDependencies

### ISSUE THREAD

**Issue by benmccann** [svelte-add/svelte-add#457]
### Topic

core

### Version

2.3.6

### Expected behavior

should be alphabetized like with a newly created project

### Actual behavior

running the tailwind adder just added them to the end:
```
	"devDependencies": {
		"@fontsource/fira-mono": "^4.5.10",
		"@neoconfetti/svelte": "^1.0.0",
		"@sveltejs/adapter-auto": "^3.0.0",
		"@sveltejs/kit": "^2.0.0",
		"@sveltejs/vite-plugin-svelte": "^3.0.0",
		"svelte": "^4.2.7",
		"svelte-check": "^3.6.0",
		"tslib": "^2.4.1",
		"typescript": "^5.0.0",
		"vite": "^5.0.3",
		"tailwindcss": "^3.4.4",
		"postcss": "^8.4.38",
		"autoprefixer": "^10.4.19"
	},
```

**Comment by benmccann** [comment]
It's still adding at the end for me

![Screenshot from 2024-07-06 08-29-04](https://github.com/svelte-add/svelte-add/assets/322311/5cc5d69f-e835-4ce2-8a2c-652a2110104a)

**Comment by manuel3108** [comment]
Cannot confirm this at this point. Seems to be working as expected for me in `2.3.6`. Which version of the cli were you using? Where you using `@latest`? I know you had some weird bugs previously which caused it to use incorrect versions of the `@svelte-add/core` package. Is this potentially happening here again?

**Comment by benmccann** [comment]
Yep, I made sure to use `@latest` and it's reporting that I'm using `2.3.6`. I created the SvelteKit demo project with default options and then added Tailwind with the default options.

> I know you had some weird bugs previously which caused it to use incorrect versions of the @svelte-add/core package. Is this potentially happening here again?

I can clear out my npx directory later and see if it helps. I wonder if we should bundle the core and config packages to avoid any chance that it gets an old version of them

**Comment by AdrianGonz97** [comment]
I'm also experiencing this as well, which is odd because I'm on the latest. I installed `svelte-add` into a project to examine the `node_modules` and I'm left even more confused lol

Checkout the `serializeJson` function in `ast-tooling` in the dist:
```js
function serializeJson(originalInput, data) {
  return patch(originalInput, data);
}
```
...it's still using `patch`, despite the version of `ast-tooling` that's also on the latest.

this could only mean one thing, a changeset wasn't made for `ast-tooling` so a new version never released 😅 I'll add one for it in #493 so that a new version is published
